sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ingestion/models/ErrorAttachmentLog.java

```diff
@@ -46,19 +44,14 @@
     private UUID errorId;
 
     /**
-     * Content type (text/plain for text).
-     */
-    private String contentType;
-
-    /**
      * File name.
      */
     private String fileName;
 
     /**
-     * Data (plain text or base64 string for binary data).
+     * Data encoded as base64.
```

Comment:
Data not encoded as base64 here. You are store data as is and encode only for json serialize.